source/renderer/app/utils/formatters.js

```diff
@@ -5,11 +5,48 @@ import {
   LOVELACES_PER_ADA,
 } from '../config/numbersConfig';
 
+// Symbol	  Name	              Scientific Notation
+// K	      Thousand	          1.00E+03
+// M	      Million	            1.00E+06
+// B	      Billion	            1.00E+09
+// T	      Trillion	          1.00E+12
+// Q	      Quadrillion	        1.00E+15
 export const formattedWalletAmount = (
   amount: BigNumber,
-  withCurrency: boolean = true
+  withCurrency: boolean = true,
+  long: boolean = false
 ) => {
-  let formattedAmount = amount.toFormat(DECIMAL_PLACES_IN_ADA);
+  let formattedAmount = '';
+  if (long) {
+    formattedAmount = amount.toFormat(DECIMAL_PLACES_IN_ADA);
+  } else if (amount.isZero()) {
+    formattedAmount = '0';
+  } else if (amount.lessThan(1)) {
+    formattedAmount = amount.round(6, BigNumber.ROUND_DOWN);
```

Comment:
@DeeJayElly I don't think we need to do any rounding down here. We have 6 decimal places anyways. Can you please remove rounding down and test?